quickstarts/microsoft.web/container-app-create/main.bicep

```diff
@@ -0,0 +1,107 @@
+@description('Specifies the name of the container app.')
+param containerAppName string = 'containerapp-${uniqueString(resourceGroup().id)}'
+
+@description('Specifies the name of the container app environment.')
+param containerAppEnvName string = 'containerapp-env-${uniqueString(resourceGroup().id)}'
+
+@description('Specifies the name of the log analytics workspace.')
+param containerAppLogAnalyticsName string = 'containerapp-log-${uniqueString(resourceGroup().id)}'
+
+@description('Specifies the location for all resources.')
+@allowed([
+  'northcentralusstage'
+  'westcentralus'
+  'eastus'
+  'westeurope'
+  'jioindiawest'
+  'northeurope'
+  'canadacentral'
+])
+param location string //cannot use resourceGroup().location since it's not available in most of regions
+
+@description('Specifie the docker container image to deploy.')
+param containerImage string = 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'
+
+@description('Specifies the container port.')
+param targetPort int = 80
+
+@description('Number of CPU cores the container can use. Can be with a maximum of two decimals.')
+param cpuCore string = '0.5'
+
+@description('Amount of memory (in gibibytes, Gi) allocated to the container up to 4Gi. Can be with a maximum of two decimals. Ratio with CPU cores must be equal to 2.')
```

Comment:
fix typo

Reply:
not sure where the typo is ? It's gibibytes GiB.